src/Form/InputClassifiersTypes.d.ts

```diff
@@ -28,6 +28,9 @@ interface InputTypeConfig {
     getIconBase: () => string,
     shouldDecorate: (boolean, InterfacePrototype) => boolean,
     dataType: 'Addresses' | 'Credentials' | 'CreditCards' | 'Identities' | '',
-    displayTitlePropName: string,
+    displayTitlePropName: (
+        input: HTMLInputElement,
+        data: CredentialsObject | IdentityObject | CreditCardObject
+    ) => string,
```

Comment:
Here, too, we're now using functions so that we can decide what to show in the tooltip based on the input field.